### DISCUSSION TITLE
Adding Prisma Migrate to an existing project

### DISCUSSION THREAD

**Discussion by audriusl** [prisma/prisma#20926] [Q&A] [answered]
### Question

So, i followed Adding Prisma Migrate to an existing project guide. The problem is that this old backend was done with migrations I think made mistake by not copying all the migrations to new project instead I used

```
npx prisma migrate diff \ 
  --from-empty \ 
  --to-schema-datamodel prisma/schema.prisma \ 
  --script > prisma/migrations/0_init/migration.sql. 
```

So, in new backend now I have `0_init` migration file. When I try to use `migrate dev` command i'm getting this error 

```
"The following migration(s) are applied to the database but missing from the local migrations directory: and the list of old my migrations from the old project". 
```

Then I copied all the migrations from the old project and then I'm getting error because migrate dev applies `0_init` first and the the old ones. Any suggestions how to get out of this situation. Database now in the production, it would be great if could avoid reseting database.

**Comment by ludralph** [comment]
Hi @audriusl 👋 

You might want to consider [Baselining](https://www.prisma.io/docs/guides/migrate/developing-with-prisma-migrate/baselining#baselining-a-database) your database as shown in the doucmentation to see if this solves the issue for you.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by audriusl** [comment]
Yes, I did that too and it is still ask for other migrations when I try "migrate dev". As I understand it is affects only "migrate deploy" command.

**Reply by ludralph** [reply]
Hi @audriusl 

Can you try following this steps?

1. Remove all migration files
2. Delete the _migrations table in the database
3. Recreate a new migration

Let me know if this helps.

**Reply by audriusl** [reply]
Hi,

How can I recreate new migration, should I just place baseline migration into the project after I deleted _migrations table in the database?

**Reply by ludralph** [reply]
To create a new migration, you would need to use the command `npx prisma migrate dev`

**Reply by ludralph** [reply]
Hey @audriusl,

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by audriusl** [reply, marked as answer]
Thank you for your help. I deleted migrations table from DB and then I followed Prisma guide for adding migrations to existing project.
